Atomic/run.py

```diff
@@ -38,6 +39,17 @@ def run(self):
             args = self._get_args("RUN")
             if args:
                 args += self.command
+                opts_file = self._get_args("RUN_OPTS_FILE")
+                if opts_file:
+                    opts_file = self.sub_env_strings("".join(opts_file))
+                    if opts_file.startswith("/"):
+                        try:
+                            run_opts = open(opts_file, "r").read()
+                            os.environ["RUN_OPTS"] = run_opts
```

Comment:
I would prefer to not modify the current environ.  We are moving away from this in another pull request.  Could we add this to self.run_opts and set it in cmd_env?